js/calc.js

```diff
@@ -0,0 +1,102 @@
+document.addEventListener('DOMContentLoaded', function() {
+
+  var buttonNum = document.getElementsByClassName('buttonNum');
+  var operator = document.getElementsByClassName('operator');
+  var buttonClear = document.getElementsByClassName('buttonClear');
+  var equals = document.getElementsByClassName('equals');
+  var screen = document.getElementById('screen');
+
+  var currentNum = '';
+  var operation = '';
+  var firstNum;
+  var secondNum;
+  var opPressed = false;
+
+  for (var i=0; i<buttonNum.length; i++) {
+    buttonNum[i].addEventListener('click',function(event) {
```

Comment:
I would separate out these functions in the click events to be actual named functions you define below, just to make the loops a little cleaner and the code a bit more abstracted and easy to understand.